src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/VisualBasicCodeRefactoringVerifier`1+Test.cs

```diff
@@ -70,12 +73,30 @@ public Test()
                 set => _sharedState.EditorConfig = value;
             }
 
+            /// <summary>
+            /// The set of code action <see cref="CodeAction.Title"/>s offered the user in this exact order.
+            /// Set this to ensure that a very specific set of actions is offered.
+            /// </summary>
+            public string[]? ExactActionSetOffered { get; set; }
```

Comment:
added these to be in sync with C# refactoring verifier (and the test updates need these).